dockerfiles/cli/scripts/cli.sh

```diff
@@ -111,11 +111,52 @@ cmd_init_reinit_pre_action() {
   fi
 }
 
+check_all_ports(){
+  docker run -d -p 80:80 \
+                -p 443:443 \
+                -p 2181:2181 \
+                -p 5000:5000 \
+                -p 23750:23750 \
+                -p 23751:23751 \
+                -p 32001:32001 \
+                -p 32101:32101 \
+                --name fake ${UTILITY_IMAGE_ALPINE} \
+                                 httpd -f -p 80 \
+                                          -p 443 \
```

Comment:
ok I think this is different on docker for mac
```
$ netstat -an | grep LISTEN
tcp4       0      0  *.8080                 *.*                    LISTEN     
```

```
docker run -p 8080:8080 -p 9000:9000  --name fb alpine:3.4 echo "hello"
c75c8218bbcd0467f9901619d7f91766e0fa82f9344de150847b2343a2538732
docker: Error response from daemon: driver failed programming external connectivity on endpoint fb (9bc1f0ac88b59c3627355031934034d78f5412bd3db231e09951f6fd61382236): Error starting userland proxy: Bind for 0.0.0.0:8080 failed: port is already allocated.
```
(no need to run as a daemon and also httpd